docs/src/examples.md

```diff
@@ -86,6 +86,24 @@ julia> round.(vcov(ols); digits=5)
   0.38889  -0.16667
  -0.16667   0.08333
 ```
+By default, the `lm` method uses the `Cholesky` factorization which is known as fast but numerically unstable, especially for ill-conditioned design matrices. You can use the `method` keyword argument to apply a more stable `QR` factorization method.
+
+```jldoctest
+julia> data = DataFrame(X=[1,2,3], Y=[2,4,7]);
```

Comment:
maybe choose an example in which Cholesky indeed does not work well?

Reply:
Sure, I will add one.